spec/lib/dsi_profile_spec.rb

```diff
@@ -41,6 +41,17 @@
           described_class.update_profile_from_dfe_sign_in dfe_user: dfe_user_no_email, local_user: provider_user
```

Comment:
I was tempted to update all the specs in this context to be in the same format
- setup
- result
- expectation
Do you think it'd be worth while?